src/components/Button/Button.stories.tsx

```diff
@@ -11,74 +11,233 @@ export default {
 
 const Template: Story<Props> = (args) => <Button {...args} />;
 
-export const Default = Template.bind({});
-Default.args = { children: 'Button' };
+export const PrimaryNoIcon = Template.bind({});
+PrimaryNoIcon.args = { children: 'Button' };
```

Comment:
nit: could we keep this `Primary`, `Secondary`, etc. (since they're the most common cases)